iothub/device/tests/Pipeline/RetryDelegatingHandlerTests.cs

```diff
@@ -454,6 +416,190 @@ public async Task RetrySetRetryPolicyVerifyInternalsSuccess()
             retryPolicy.Counter.Should().Be(2);
         }
 
+        [Ignore]
```

Comment:
Marking this test as ignored for now since this functionality hasn't been added yet